extjsdk/src/main/java/io/vantiq/extjsdk/Utils.java

```diff
@@ -1,114 +1,127 @@
 package io.vantiq.extjsdk;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.util.Properties;
 
 public class Utils {
 
+    // String used by methods to synch on
+    private static final String SYNCH_LOCK = "sycnhLockString";
+
     public static final String SEND_PING_PROPERTY_NAME = "sendPings";
     public static final String PORT_PROPERTY_NAME = "tcpProbePort";
     public static final String SERVER_CONFIG_DIR = "serverConfig";
     public static final String SERVER_CONFIG_FILENAME = "server.config";
     public static final String SECRET_CREDENTIALS = "CONNECTOR_AUTH_TOKEN";
 
-    private static final Logger log = LoggerFactory.getLogger(Utils.class);
+    // The properties object containing the data from the server configuration file
```

Comment:
Probably want to mark this as `@VisibleForTesting` since there's no reason here to expose this?  Or make it private (so sync protocol is guaranteed) and change the test to test in some other way (which is probably a better solution) -- rewrite the file or something to make sure you get the new stuff, clear the file & get nothing, etc.